src/helpers/assets.js

```diff
@@ -47,7 +46,7 @@ export const buildCoinsList = (
 
   const isShortList = assetsLength <= amountOfShowedCoins;
 
-  forEach(assets, asset => {
+  forEach(assets, function(asset, index) {
```

Comment:
you don't need to add `function`, just keep the arrow and add parenthesis 

` forEach(assets, (asset, index) => {`